### PR TITLE
Feature/bridges mpirun

### PR DESCRIPTION
**Pull request by andre-merzky**
add an mpirun config for bridges

### CODE REVIEW

**Comment by iparask** [comment]
Is this for Repex?

**Comment by andre-merzky** [comment]
This was originally targeting repex, but is really just a generic mpirun config for that machine.

File: src/radical/pilot/configs/resource_xsede.json [hunk 1]
```diff
@@ -552,6 +552,52 @@
...
+        },
+        "default_queue"               : "RM",
+        "resource_manager"            : "SLURM",
+        "lfs_path_per_node"           : "$LOCAL",
+        "lfs_size_per_node"           : 3713368,
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "agent_launch_method"         : "MPIEXEC",
+        "task_launch_method"          : "MPIEXEC",
+        "mpi_launch_method"           : "MPIEXEC",
```
Comment (iparask) [inline src/radical/pilot/configs/resource_xsede.json:583]:
```suggestion
        "agent_spawner"               : "POPEN",
        "agent_launch_method"         : "SSH",
        "task_launch_method"          : "SSH",
        "mpi_launch_method"           : "MPIEXEC",
```

May I suggest this change then? For some reason the agent did not start with MPIEXEC.

Comment (andre-merzky) [reply src/radical/pilot/configs/resource_xsede.json:583]: I would like to keep `MPIEXEC` for the `task_launcher` though, as it does not suffer from saturation issues like `SSH`.

Comment (andre-merzky) [reply src/radical/pilot/configs/resource_xsede.json]: That is pushed now.  Having said that, do you understand why MPIEXEC failed for the agent?  Do you still happen to have the logs around?

Comment (iparask) [reply src/radical/pilot/configs/resource_xsede.json]: I did not pay too much attention as the purpose was to get repex running. Unfortunately, I do not have the logs around anymore. I can generate them if you want.

Comment (andre-merzky) [reply src/radical/pilot/configs/resource_xsede.json]: Nah, don't worry - we have enough on our table...  Lets handle that when it pops up again...

**Review by iparask — APPROVED** [review]

File: src/radical/pilot/configs/resource_xsede.json [hunk 2]
```diff
@@ -552,6 +552,52 @@
...
+                                         "module load mpi/intel_openmpi/19.3"
+                                        ],
+        "default_remote_workdir"      : "$SCRATCH",
+        "valid_roots"                 : ["/home", "/pylon1", "/pylon5"],
+        "rp_version"                  : "local",
+        "virtenv_mode"                : "create",
+        "python_dist"                 : "anaconda",
+        "export_to_cu"                : [],
+        "cu_pre_exec"                 : [],
+        "saga_jd_supplement"          : {"cpu_architecture" : ["p100"]},
```
Comment (mtitov) [inline src/radical/pilot/configs/resource_xsede.json:597]: @andre-merzky this line is obsolete ... should be `"system_architecture"         : {"gpu": "p100"}`

**Pull request merged**